engines/bladerunner/ui/vk.cpp

```diff
@@ -683,10 +690,10 @@ void VK::drawNeedle(Graphics::Surface &surface) {
 	int g =  8 * colorIntensity;
 	int b = 12 * colorIntensity;
 
-	surface.drawLine(203, 324, x - 2, y,     ((7 - r    ) << 10) | ((18 - g    ) << 5) | (23 - b    ));
```

Comment:
some of whitespace removal is ok but some, like this, are reducing readibility

Reply:
This was AStyle. I'm sorry it ended messing some stuff up. I'll try to revert all of the style changes.